ALUM/ALUM/Views/ContentView.swift

```diff
@@ -9,9 +9,34 @@ import SwiftUI
 
```

Comment:
@YashRavipati1 Small note! Can we remove the updates from ContentView because it is easy for these to be left out. Let's use FirebaseContentView in ALUMApp instead.